frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -909,7 +909,7 @@
             vm.remainingTime = {};
             vm.showClock = false;
             vm.showSubmissionNumbers = false;
-            parameters.url = "jobs/"+ vm.challengeId + "/phases/"+ vm.phaseId + "/remaining-submissions/";
+            parameters.url = "jobs/"+ vm.challengeId + "/phases/"+ vm.phaseId + "/remaining_submissions/";
```

Comment:
@the-dagger can you please remove this since this is not the part of the error that you are solving here. It would be better if this comes as a separate pull request.